app/controllers/orders_controller.rb

```diff
@@ -11,7 +11,7 @@ def edit
     def update
         @order = current_order
         if @order.update(order_params)
-            @order = current_order
+            @order.update(currency: current_order.products.first.currency)
```

Comment:
Currency of the web shop - I guess we can assume that it's one currency - can be hard-coded in a constant in `config/initializers/constants.rb`

Reply:
Added has a hard-code and changed